### PR TITLE
Fix all linking errors and address isue 439

### PR DESCRIPTION
**Pull request by AngeloKai**
The following is to address linking errors in the spec to publish WD-05.

***
[Preview](https://s3.amazonaws.com/pr-preview/AngeloKai/webauthn/WD05.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/webauthn/b7aac16...AngeloKai:f4bf910.html)

### CODE REVIEW

**Comment by AngeloKai** [comment]
Below are the changes: 
- Change all RP instances to [=[RP]=] to fix a linking error. 
- Change {{Credential/[[discovery]]}}  to <dfn>\[[discovery]]</dfn>. This is to be consistent with the `type` change. 
- Change |attachment| to {{AuthenticatorSelectionCriteria/attachment}} and |requireResidentKey| to {{AuthenticatorSelectionCriteria/attachment}}. This fixes another linking error.

**Comment by AngeloKai** [comment]
@leshi, can you approve this as well?

**Comment by AngeloKai** [comment]
@jyasskin, can you review this? Or otherwise I'd just merge it.

File: index.bs [hunk 1]
```diff
@@ -1067,9 +1067,8 @@ associated.
 
     :   <dfn>requireResidentKey</dfn>
     ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=Client-side-resident Credential 
-        Private Key=]. If the parameter is set to <a>true</a>, the authenticator MUST create a 
-        [=Client-side-resident Credential Private Key=] when creating a [=scoped credential=].
-
+        Private Key=]. If the parameter is set to true, the authenticator MUST create a
```
Comment (jyasskin) [inline index.bs]: You should use "<code>&#x60;true&#x60;</code>" to typeset this, but again, the text you have isn't terrible.

**Pull request merged**